pkg/utils/openstack/loadbalancer.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package openstack
+
+import (
+	version "github.com/hashicorp/go-version"
+	klog "k8s.io/klog/v2"
+)
+
+const (
+	OctaviaFeatureTags              = 0
+	OctaviaFeatureVIPACL            = 1
+	OctaviaFeatureFlavors           = 2
+	OctaviaFeatureTimeout           = 3
+	OctaviaFeatureAvailabilityZones = 4
+	lbProviderOVN                   = "ovn"
+)
+
+// IsOctaviaFeatureSupported returns true if the given feature is supported in the deployed Octavia version.
+// copied from https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/util/openstack/loadbalancer.go#L95-L148
+func IsOctaviaFeatureSupported(octaviaVer string, feature int, lbProvider string) bool {
+	currentVer, _ := version.NewVersion(octaviaVer)
+
+	switch feature {
+	case OctaviaFeatureVIPACL:
+		if lbProvider == lbProviderOVN {
+			return false
+		}
```

Comment:
We can set the `lbProviderOVN` validation right before the switch-case to fail early (even if the compiler might do this anyway).

Reply:
From the copied feature-matrix from https://github.com/kubernetes/cloud-provider-openstack/blob/master/pkg/util/openstack/loadbalancer.go#L95-L148 tags are also supported on OVN

```go
	case OctaviaFeatureTags:
		verTags, _ := version.NewVersion("v2.5")
		if currentVer.GreaterThanOrEqual(verTags) {
			return true
		}
```

Reply:
Sure, totally missed this 👍🏻